OWNERS

```diff
@@ -5,17 +5,16 @@ filters:
       - davidvossel
       - dhiller
       - enp0s3
-      - fgimenez
       - phoracek
       - rmohr
       - tiraboschi
       - vladikr
+      - brianmcarey
     approvers:
       - aglitke
       - davidvossel
       - dhiller
       - enp0s3
-      - fgimenez
```

Comment:
@brianmcarey would you mind moving Federico to the `emeritus_approvers` section?

Reply:
yes I will make that change

Reply:
Thanks! :+1: